### PR TITLE
Working endpoints to create and get reservations.

### PR DESCRIPTION
**Pull request by stevkim**
Currently a lot of the logic is based on whether the front-end takes care of data structure. 
Will need to come back to refactor once everything is set.

### CODE REVIEW

File: server/routes/reservationsRouter.js [hunk 1]
```diff
@@ -21,5 +29,16 @@ reservationRouter.route('/test')
     }
   })
 
+reservationRouter.route('/:garage_id')
+  .get(async (req, res) => {
+    console.log(req.params, req.query);
```
Comment (kjmak14) [inline server/routes/reservationsRouter.js:34]: Not sure if you meant to leave this console log in for the PR

**Review by kjmak14 — APPROVED** [review]
Looks good, the logic is sound and I like the way you organized the logic and routes.  I think Kyle S. and I will be able to use the getAllReservations controller for our widget so thanks!

File: server/routes/reservationsRouter.js (hunk 1 above)
Comment (stevkim) [reply server/routes/reservationsRouter.js:34]: Gotcha, i'll remove this!

**Review by clee4037 — APPROVED** [review]
Looks good! Idk what else to say lol.... Looking forward to testing it.

**Pull request merged**